contrib/builder-keys/README.md

```diff
@@ -19,9 +19,16 @@ gpg --refresh-keys
 To fetch keys of builders and active developers, feed the list of fingerprints
 of the primary keys into gpg:
 
+using \*NIX:
```

Comment:
nit: it's more usual to say "on UNIX" and "on Windows", instead of "using"

Reply:
Accepted and applied.